.circleci/config.yml

```diff
@@ -140,6 +143,7 @@ jobs:
       - acctest:
           provider-test-infra-dir: k8s
           provider-go-test-dir: k8s
+          provider-tf-apply: "terraform apply -target google_container_cluster.cluster -auto-approve && terraform apply -auto-approve"
```

Comment:
Can you add a small comment above this as to why this is necessary? a link to the terraform doc would be fine https://www.terraform.io/docs/providers/kubernetes/index.html#stacking-with-managed-kubernetes-cluster-resources

Reply:
✅